metabase/metabase
# Do not autorun ad-hoc native queries

How to test:
- New -> Question -> SQL query
- Make sure you see `Here's where your results will appear`

<img width="716" alt="Screenshot 2022-09-28 at 15 53 27" src="https://user-images.githubusercontent.com/8542534/192783501-15136259-e83b-4325-9c3b-4425e77bc6c3.png">
